ports/rp2/boards/PICO_W/pins.csv

```diff
@@ -0,0 +1,13 @@
+WL_GPIO0,EXT_GPIO0
```

Comment:
The Pico W only has 3 external pins.

```c
#define CYW43_WL_GPIO_COUNT 3
```

Reply:
Right :man_facepalming:  sorry

Reply:
FWIW this is `board` table with only 3 pins:
```c
STATIC const mp_rom_map_elem_t pin_board_pins_locals_dict_table[] = {
  #if (MICROPY_HW_PIN_EXT_COUNT > 0)
  { MP_ROM_QSTR(MP_QSTR_WL_GPIO0), MP_ROM_PTR(&pin_EXT_GPIO0) },
  #endif
  #if (MICROPY_HW_PIN_EXT_COUNT > 1)
  { MP_ROM_QSTR(MP_QSTR_WL_GPIO1), MP_ROM_PTR(&pin_EXT_GPIO1) },
  #endif
  #if (MICROPY_HW_PIN_EXT_COUNT > 2)
  { MP_ROM_QSTR(MP_QSTR_WL_GPIO2), MP_ROM_PTR(&pin_EXT_GPIO2) },
  #endif
  #if (MICROPY_HW_PIN_EXT_COUNT > 0)
  { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_EXT_GPIO0) },
  #endif
};
MP_DEFINE_CONST_DICT(pin_board_pins_locals_dict, pin_board_pins_locals_dict_table);
```